keras/run-keras-notebook.sh

```diff
@@ -17,7 +17,7 @@ start=$(date "+%s")
 ${ZOO_TUTORIALS}/keras/ipynb2py.sh ${ZOO_TUTORIALS}/keras/2.1-a-first-look-at-a-neural-network
 
 sed "s/nb_epoch=5/nb_epoch=5/g" ${ZOO_TUTORIALS}/keras/2.1-a-first-look-at-a-neural-network.py >${ZOO_TUTORIALS}/keras/tmp_test.py
-#sed -i "s/get_ipython()/#/g" ${ZOO_TUTORIALS}/keras/tmp_test.py
+sed -i "s/zoo.pipeline.api.keras.datasets/tensorflow.keras.datasets/g" ${ZOO_TUTORIALS}/keras/tmp_test.py
```

Comment:
I did this because loading data from `zoo.pipeline.api.keras.datasets` results in "403 forbidden" too frequently (data source is shown here: https://github.com/intel-analytics/analytics-zoo/blob/master/pyzoo/zoo/pipeline/api/keras/datasets/mnist.py#L23), and loading data from `tensorflow.keras.datasets` is much more stable in the tests. Is this change ok?

Reply:
I think its ok!